<ui version="4.0" >
 <class>AudioSettings</class>
 <widget class="QWidget" name="AudioSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>519</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Devices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="1" column="2" >
         <widget class="QComboBox" name="outputDeviceComboBox" />
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Input Device:</string>
          </property>
          <property name="buddy" >
           <cstring>inputDeviceComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Output Device:</string>
          </property>
          <property name="buddy" >
           <cstring>outputDeviceComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/audio/speaker.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/audio/speaker.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="label_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Ringing Device:</string>
          </property>
          <property name="buddy" >
           <cstring>ringingDeviceComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QComboBox" name="ringingDeviceComboBox" />
        </item>
        <item row="0" column="2" >
         <widget class="QComboBox" name="inputDeviceComboBox" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/audio/microphone.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <layout class="QHBoxLayout" />
        </item>
        <item row="1" column="3" >
         <widget class="QPushButton" name="testOutputDeviceButton" >
          <property name="text" >
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QPushButton" name="testRingingDeviceButton" >
          <property name="text" >
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Security</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="0" colspan="4" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>VoxOx cans encrypt calls using the AES 128-bits encryption system and Diffie-Hellman for key exchange.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="callEncryptionCheckBox" >
        <property name="text" >
         <string>Enable call encryption</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>356</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="4" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Note: Your contacts need to enable this option for the call to be encrypted. To know whether this call is indeed encrypted, please check on the bottom of your VoxOx window</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="halfDuplexCheckBox" >
          <property name="text" >
           <string>&amp;Half Duplex</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3" >
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="1" >
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Advanced - Beware, this section is for power users only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item rowspan="2" row="5" column="0" colspan="2" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <item rowspan="3" row="0" column="0" >
         <widget class="QListWidget" name="audioCodecListWidget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>150</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="UpButton" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../../resources/wengophone.qrc" >
            <normaloff>:/pics/up.png</normaloff>:/pics/up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="DownButton" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../../resources/wengophone.qrc" >
            <normaloff>:/pics/down.png</normaloff>:/pics/down.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QPushButton" name="ResetButton" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text" >
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="2" colspan="2" >
       <widget class="QLabel" name="label_7" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="text" >
         <string>Any change in the order of your codec list may cause problems running VoxOx; you might not be able to call your contacts anymore.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>17</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputDeviceComboBox</tabstop>
  <tabstop>outputDeviceComboBox</tabstop>
  <tabstop>testOutputDeviceButton</tabstop>
  <tabstop>ringingDeviceComboBox</tabstop>
  <tabstop>testRingingDeviceButton</tabstop>
  <tabstop>audioCodecListWidget</tabstop>
  <tabstop>UpButton</tabstop>
  <tabstop>DownButton</tabstop>
  <tabstop>ResetButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
  <include location="../../../../resources/wengophone.qrc" />
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
